services/business/istio_config.go

```diff
@@ -35,16 +36,19 @@ func (in *IstioConfigService) GetIstioConfig(criteria IstioConfigCriteria) (mode
 		DestinationPolicies: models.DestinationPolicies{},
 		VirtualServices:     models.VirtualServices{},
 		DestinationRules:    models.DestinationRules{},
+		ServiceEntries:      models.ServiceEntries{},
 		Rules:               models.IstioRules{},
 	}
-	var ggErr, rrErr, dpErr, vsErr, drErr, mrErr error
+	var gg, rr, dp, vs, dr, se []kubernetes.IstioObject
```

Comment:
just out of curiosity, why declaring all of them here if they're only used in their goroutine's scope?

Reply:
Basically to not duplicate a local variable inside the goroutine.
Internally it was using gg, ggErr := ... now it is changed to gg, ggErr = ... without creating the variable which is tested outside the goroutine.
It was missing some errors.

Reply:
ah yes, makes sense!